package.json

```diff
@@ -50,6 +51,7 @@
     "eslint-plugin-react": "^7.29.4",
     "eslint-webpack-plugin": "^3.1.1",
     "fork-ts-checker-webpack-plugin": "^7.2.1",
+    "html2canvas": "^1.4.1",
```

Comment:
Not used. Please remove

Reply:
Not used, but vite dev server requires it to be found&hellip; It will show error if not found this module.

Reply:
Maybe we need to use: `peerDependencies`?

Reply:
Maybe I could just update the version of white-snapshot to remove html2canvas support.

Reply:
done.